.gitattributes

```diff
@@ -0,0 +1,8 @@
+/.github/           export-ignore
+/docs/              export-ignore
+/tests/             export-ignore
+/.gitattributes     export-ignore
+/.gitignore         export-ignore
+/build.xml          export-ignore
+/phpcs.xml          export-ignore
+/phpunit.xml.dist   export-ignore
```

Comment:
None of the directories and files are required by anyone consuming this package as a dependency with `composer`.